codebrag-dao/src/main/scala/com/softwaremill/codebrag/dao/finders/followup/FollowupFinder.scala

```diff
@@ -9,9 +9,13 @@ trait FollowupFinder {
   def findAllFollowupsByCommitForUser(userId: ObjectId): FollowupsByCommitListView
 
   def findFollowupForUser(userId: ObjectId, followupId: ObjectId): Either[String, SingleFollowupView]
+  
+  def findFollowupforAdmin(followupId: ObjectId): Either[String, SingleFollowupView]
```

Comment:
this function should be renamed to `findFollwup` - we do not restrict to admins anymore, right?